lib/shared/api_database.dart

```diff
@@ -70,6 +70,14 @@ class ApiDatabase {
     String addressId = "0/0";
     Map<String, dynamic> addressList = {'$currentWalletId': '0/0'};
 
+    // if currentWalletId exists, preferences should be re-written in local storage
+    if (currentWalletId != null) {
+      await setWalletAndAccountInLocalStorage(
+          currentWalletId,
+          AddressEntry(
+              walletId: currentWalletId, addressIdx: addressId, keyType: 0));
+    }
+
```

Comment:
Could we remove this? It's already included in https://github.com/witnet/my-wit-wallet/pull/259
```suggestion
```